zrml/court/src/lib.rs

```diff
@@ -122,20 +164,29 @@ mod pallet {
         T: Config,
     {
         type AccountId = T::AccountId;
+        type Balance = BalanceOf<T>;
         type BlockNumber = T::BlockNumber;
         type Origin = T::Origin;
         type MarketId = MarketIdOf<T>;
 
-        fn on_dispute(
-            _origin: Self::Origin,
+        fn on_dispute<D>(
+            dispute_bond: D,
             _market_id: Self::MarketId,
             _outcome: OutcomeReport,
-        ) -> Result<[u32; 2], DispatchError> {
-            todo!()
+            who: Self::AccountId,
+        ) -> DispatchResult
+        where
+            D: Fn(usize) -> Self::Balance,
+        {
+            CurrencyOf::<T>::reserve(&who, dispute_bond(1))?;
+            let jurors: Vec<_> = Jurors::<T>::iter().collect();
+            let _ = Self::random_jurors(&jurors, 3);
```

Comment:
Can we make `3` a constant or value within the pallet's config?

Reply:
This matter is already addressed on https://github.com/zeitgeistpm/zeitgeist/pull/233